ERCS/erc-7690.md

```diff
@@ -0,0 +1,129 @@
+---
+eip: 7690
+title: Union Lock
+description: Union Lock based on TSTORE/TLOAD can avoid flash loan attacks.
```

Comment:
The most important thing your description should explain is probably what a "union lock" is.

Reply:
change it to "Union Lock prevent these functions or agreements from being accessed a second time or multiple times by recording the number of calls to a certain function or the number of times a certain agreement is called in a transaction. When the second or multiple accesses are called, Reject the transaction immediately."